js/gulp/compile-task.js

```diff
@@ -25,12 +25,12 @@ const typescriptTask = require('./typescript-task');
 const { arrowTask, arrowTSTask } = require('./arrow-task');
 
 const compileTask = ((cache) => memoizeTask(cache, function compile(target, format, ...args) {
-    return target === `src`                    ? Observable.empty()
-         : target === npmPkgName               ? arrowTask(target, format, ...args)()
-         : target === `ts`                     ? arrowTSTask(target, format, ...args)()
-         : format === `umd` ? target === `es5` ? closureTask(target, format, ...args)()
-                                               : minifyTask(target, format, ...args)()
-                                               : typescriptTask(target, format, ...args)();
+    return target === `src`                       ? Observable.empty()
+         : target === npmPkgName                  ? arrowTask(target, format, ...args)()
+         : target === `ts`                        ? arrowTSTask(target, format, ...args)()
+         : format === `umd` ? target === `es2015` ? closureTask(target, format, ...args)()
```

Comment:
@trxcllnt Does this look right to you?

Reply:
Does Closure finally produce good ES2015+? I think we've only been using it to make the ES5 UMD bundle because it either was bad, or didn't always work to compile ES2015+.

If it does, I'd be happy to rip out Webpack + Terser and use Closure for all the UMD bundles.

Reply:
I don't know (closure compiler is a mystery to be tbh). I thought this may be needed to switch to es2015 but I'll undo this for now.

Reply:
Done

Reply:
Closure compiler is a mystery to me as well :joy:. I have to doc-dive every time we update to find the new combinations of magic strings should be. lol yuck